init-script

```diff
@@ -337,15 +337,22 @@ if [ "$DONE_SWITCH" = "no" ]; then
 	usb_setup "Mer Debug: done debug, disabling storage"
     fi
 
+    # Remount the target as ro if it provides the .halium-ro file
+    if [ -e /target/.halium-ro ]; then
+        mount /target -o remount,ro
```

Comment:
Any reason you are doing remount, instead of mounting as ro from start only?

Reply:
Some versions of Busybox will make the loop device itself (as well as the mount point) read-only if ro is specified at mount time. That means the image can't easily be remounted rw. Remounting like this means we have a read-only mount point but a read-write loop. 

Ref: https://github.com/ubports/ubports-boot/blob/fd906a4ddaff887a2d87dd0b37277e452abe1266/initramfs/scripts/touch#L337